lib/puppet/indirector/file_server.rb

```diff
@@ -15,7 +15,15 @@ def authorized?(request)
 
     # If we're not serving this mount, then access is denied.
     return false unless mount
-    mount.allowed?(request.node, request.ip)
+
+    # If there are no auth directives or there is an 'allow *' directive, then
+    # access is allowed.
+    if mount.empty? || mount.globalallow?
```

Comment:
@camlow325 I disagree here - IMO we should retain the previous behavior of denying if no allow/deny directives are present. I think the only way access should be granted is if they've done `allow *`. Admittedly I don't know what the use case would be for having such a mount defined, but maybe one exists?